shell/browser/native_window_mac.mm

```diff
@@ -1288,7 +1288,11 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
     const bool no_rounded_corner =
         [window_ styleMask] & NSWindowStyleMaskFullSizeContentView;
     if (!has_frame() && !is_modal() && !no_rounded_corner) {
-      CGFloat radius = 5.0f;  // default corner radius
+      if (@available(macOS 11, *)) {
+        CGFloat radius = 9.0f;
+      } else {
```

Comment:
needs to be declared outside this conditional:

```cpp
CGFloat radius;
if (@available(macOS 11.0, *)) {
   radius = 9.0f;
} else {
   radius = 5.0f;
}
```

Reply:
Ah thanks for the help!